src/renderer/i18n/de/error.ts

```
@@ -0,0 +1,8 @@
+import { ErrorMessages } from '../types'
+
+const error: ErrorMessages = {
+  'error.chainflip.channelIdRequired': 'Kanal-ID für Chainflip-Tracking erforderlich',
+  'error.interact.missingData.title': 'Fehlende Daten für InteractiveView'
+}
+
+export default error
```

ESLint

[error] Insert `⏎`

(prettier/prettier)